### ISSUE TITLE
Build failure under Alpine Linux

### ISSUE THREAD

**Issue by FlushablPet** [lijunle/Vsxmd#76]
Hi, first off, great tool, I love it.

It looks like it doesn't have Alpine compatible binaries though, as it caused our CI pipeline's build stage to fail when used with the Dotnet Core Alpine image `mcr.microsoft.com/dotnet/core/sdk:3.1-alpine`. Switching to an Ubuntu flavor worked, however adding Alpine support to the package for those working in lightweight CI environments would be a bonus as Alpine continues to increase in popularity.

```bash
/root/.nuget/packages/vsxmd/1.4.4/build/Vsxmd.targets(16,5): error MSB3073: The command ""/root/.nuget/packages/vsxmd/1.4.4/build//../tools/linux-x64/Vsxmd" "Wyatt.Data.xml" "" """ exited with code 127.
```

**Comment by lijunle** [comment]
Hi, @FlushablPet 

For sure that, I could like to support alpine. Let me try if I could repro it. If it is a x64 distribution, it should work.

**Comment by lijunle** [comment]
I can repro it and identify the cause. Apline does not belong to `linux-x64` group, but `linux-musl-x64`. See [here](https://github.com/dotnet/core/issues/1689#issuecomment-397032261).

I will include that into build and fix this issue.

**Comment by lijunle** [comment]
@FlushablPet 

I published the version [1.4.5-beta.1](https://www.nuget.org/packages/Vsxmd/1.4.5-beta.1) on addressing this. Please help on testing it and let me know. Thanks a lot on reporting this!

**Comment by FlushablPet** [comment]
Just tested it in the CI and Docker builds, both worked like a charm. Thanks so much for adding it so quickly!

**Comment by lijunle** [comment]
Thanks for helping testing! I merged my changes to master and published [`v1.4.5`](https://www.nuget.org/packages/Vsxmd/1.4.5) for it. Again, thank you for reporting this issue!
